crates/vm/db.rs

```diff
@@ -1,8 +1,80 @@
-use ethrex_common::types::BlockHash;
+use bytes::Bytes;
+use ethrex_common::{
+    types::{AccountInfo, BlockHash, ChainConfig},
+    Address, H256, U256,
+};
 use ethrex_storage::Store;
 
+use crate::EvmError;
+
+#[derive(Clone)]
+pub struct VmDbWrapper<T: VmDatabase>(pub T);
```

Comment:
This is required since the idea is to move `StoreWrapper` outside of the vm crate and into blockchain. But I need to have a type defined inside vm that implements LevmDatabase & RevmDatabase.